### ISSUE TITLE
getPhoneNumber() undefined 

### ISSUE THREAD

**Issue by suskind** [rebeccahughes/react-native-device-info#223]
is it possible to get phone number in Android? 

dumping object, it doesn't have `getPhoneNumber` method

```
I/ReactNativeJS(31439): 'DEBUG', { getUniqueID: [Function: getUniqueID],
I/ReactNativeJS(31439):   getInstanceID: [Function: getInstanceID],
I/ReactNativeJS(31439):   getDeviceId: [Function: getDeviceId],
I/ReactNativeJS(31439):   getManufacturer: [Function: getManufacturer],
I/ReactNativeJS(31439):   getModel: [Function: getModel],
I/ReactNativeJS(31439):   getBrand: [Function: getBrand],
I/ReactNativeJS(31439):   getSystemName: [Function: getSystemName],
I/ReactNativeJS(31439):   getSystemVersion: [Function: getSystemVersion],
I/ReactNativeJS(31439):   getBundleId: [Function: getBundleId],
I/ReactNativeJS(31439):   getBuildNumber: [Function: getBuildNumber],
I/ReactNativeJS(31439):   getVersion: [Function: getVersion],
I/ReactNativeJS(31439):   getReadableVersion: [Function: getReadableVersion],
I/ReactNativeJS(31439):   getDeviceName: [Function: getDeviceName],
I/ReactNativeJS(31439):   getUserAgent: [Function: getUserAgent],
I/ReactNativeJS(31439):   getDeviceLocale: [Function: getDeviceLocale],
I/ReactNativeJS(31439):   getDeviceCountry: [Function: getDeviceCountry],
I/ReactNativeJS(31439):   getTimezone: [Function: getTimezone],
I/ReactNativeJS(31439):   isEmulator: [Function: isEmulator],
I/ReactNativeJS(31439):   isTablet: [Function: isTablet],
I/ReactNativeJS(31439):   isPinOrFingerprintSet: [Function: isPinOrFingerprintSet] }

``` 

or am I doing it wrong? 

Thank you

**Comment by GantMan** [comment]
What version are you currently using?

**Comment by suskind** [comment]
Of what? react-native or this module?

react-native-cli: 2.0.1
react-native: 0.48.3

react-native-device-info: 0.11.0

**Comment by hemedani** [comment]
I have the same issue
```
 "react-native": "0.47.2",
 "react-native-device-info": "^0.11.0",
```

**Comment by hemedani** [comment]
I'm also checked on real phone and error is still there

**Comment by saalihou** [comment]
It looks like the latest npm release (0.11.0) does not ship with the method (I checked the node_modules folder). I installed from the master branch on github directly. You can try that out (I use yarn so `yarn add yarn add https://github.com/rebeccahughes/react-native-device-info.git`, should be almost the same with npm).

**Comment by GantMan** [comment]
Awaiting 1 last feature and shipping

**Comment by afilp** [comment]
I am in latest version (0.12) and while debugging with my real device, I do not get my phone number (Android). Is this working?

**Comment by tehong** [comment]
I'm on 0.13.0 and my RN Android DeviceInfo.getPhoneNumber() returned "undefined" as well.   It looks to me it hasn't been implemented yet.  Is that correct?

**Comment by tehong** [comment]
It looks like there is no reliable way to get phone number in android based on [https://stackoverflow.com/questions/35167273/how-to-get-phone-number-of-the-device-in-android-activity](url)

**Comment by machour** [comment]
This orignal issue was fixed in 0.12.0
